src/app/_components/home/activity/activity-home.css

```
@@ -189,58 +189,167 @@
+  font-weight: 500;
 }
 
-@media (max-width: 920px) {
+/* ---- actions ---- */
+.wsa-week__cta-row {
+  display: flex;
+  gap: 10px;
+  flex-wrap: wrap;
+}
+
+@media (max-width: 860px) {
```

Stylelint (17.12.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)